scripts/app

```diff
@@ -88,6 +88,7 @@ compose() {
 
   export BITCOIN_DATA_DIR="${UMBREL_ROOT}/bitcoin"
   export LND_DATA_DIR="${UMBREL_ROOT}/lnd"
+  export TOR_DATA_DIR="${UMBREL_ROOT}/tor/data"
```

Comment:
Why is this exported?

We shouldn't allow apps to mount the Tor data dir, even as read only, it contains the private keys for all Tor hidden services.

Reply:
Following a discussion on Keybase, this is not needed and will be removed.